### PR TITLE
users password pdf

### PR DESCRIPTION
**Pull request by ostcar**
Fixes: #1496

### CODE REVIEW

File: tests/integration/users/test_views.py [hunk 1]
```diff
@@ -87,3 +87,17 @@ def test_post_incorrect_data(self):
...
+        """
+        Tests that the view returns the status code 200.
+        """
+        self.client.login(username='admin', password='admin')
+        response = self.client.get('/users/passwords/print/')
+
+        self.assertEqual(
+            response.status_code,
+            200,
+            "The status code of the user password view should be 200")
```
Comment (normanjaeckel) [inline tests/integration/users/test_views.py]: `... user password PDF view ... 200.`

**Comment by normanjaeckel** [comment]
Please add a more explicit commit message (with capital letter if you like :-), have a look at my comment, rebase and merge.

File: tests/integration/users/test_views.py (hunk 1 above)
Comment (ostcar) [inline tests/integration/users/test_views.py]: done

**Comment by ostcar** [comment]
Travis fails, because it can not load the development version of coverage. I do not understand way. @normanjaeckel can you find a solution?

**Comment by normanjaeckel** [comment]
Done via restart of Travis Job.

**Comment by ostcar** [comment]
I tried this...

can you merge this pull request?

**Pull request merged**